projects/control-service/projects/model/apidefs/datajob-api/api.yaml

```diff
@@ -900,6 +900,57 @@ paths:
         in: path
         required: true
 
+  '/data-jobs/for-team/{team_name}/jobs/{job_name}/deployments/{deployment_id}/secrets':
+    summary: |
+      Data Job Secrets API.
```

Comment:
Try to prove more information to users what they can used the api for . In which cases they can use properties vs secrets api. What they provide in terms of guarantees

Reply:
expanded section.